packages/console/src/browser/ansi-console-item.tsx

```diff
@@ -17,10 +17,7 @@
 import * as React from 'react';
 import { ConsoleItem } from './console-session';
 import { Severity } from '@theia/core/lib/common/severity';
-
-// TODO: Remove this workaround once issue is fixed by anser.
-// REF: https://github.com/IonicaBizau/anser/issues/52
-const { ansiToHtml } = (require('anser') as import('anser').default);
+import Anser from 'anser';
```

Comment:
`import Anser = require('anser')` is better in this case, no more need for `esModuleInterop`.

Reply:
Fixed.